src/tmpfiles/tmpfiles.c

```diff
@@ -3973,16 +3973,16 @@ static int help(void) {
         if (r < 0)
                 return log_oom();
 
-        printf("%s [OPTIONS...] [CONFIGURATION FILE...]\n"
-               "\n%sCreates, deletes and cleans up volatile and temporary files and directories.%s\n\n"
+        printf("%1$s [OPTIONS...] [CONFIGURATION FILE...]\n"
+               "\n%2$sCreates, deletes and cleans up volatile and temporary files and directories.%3$s\n\n"
                "  -h --help                 Show this help\n"
                "     --user                 Execute user configuration\n"
                "     --version              Show package version\n"
                "     --cat-config           Show configuration files\n"
                "     --tldr                 Show non-comment parts of configuration\n"
-               "     --create               Create marked files/directories\n"
-               "     --clean                Clean up marked directories\n"
-               "     --remove               Remove marked files/directories\n"
+               "     %2$s--create%3$s               Create files and directories\n"
+               "     %2$s--clean%3$s                Clean up files and directories\n"
+               "     %2$s--remove%3$s               Remove files and directories\n"
```

Comment:
I'd prefer i we seperate this out by sections, independently of color.

i.e distinguish between "option" switches, and "command" switches, and but those in separet sections, but the relevant commands first.

Or alternatively, add two command sections "Primary Commands" and just "Commands" for the rest.

Reply:
Or "Operations" rather then "Primary COmmands"?